### DISCUSSION TITLE
Error message enquiry

### DISCUSSION THREAD

**Discussion by insighttrainingbackup** [ptarmiganlabs/butler-sheet-icons#205] [General]
Hi

When I run the in Windows powershell, I get the following error

CLOUD APP: Error: Failed to launch the browser process! spawn C:\Users\JANHEN~1\AppData\Local\Temp\.local-chromium\win64-961656\chrome-win\chrome.exe ENOENT

Do you maybe know what I need to do to avoid this error?
![Ashampoo_Snap_Thursday, March 31, 2022_02h09m48s_018_](https://user-images.githubusercontent.com/99006717/160950775-57d91ea4-33f0-4ff8-b5e7-db52efb937ad.png)

Thanks so much

**Comment by mountaindude** [comment]
@insighttrainingbackup 
Looong overdue response here..

I have tried various things to reproduce this error, but on the Windows computers here (Win10, WinSrv2016, WinSrv2019) things work as expected. 
Lots of updates and changes have been done to Butler Sheet Icons during past months, so it might be worth downloading the latest version and trying again (in case you never got it working back in March -22).
